src/main/java/de/siphalor/modsoftheworld/client/MOTWClient.java

```diff
@@ -78,7 +84,7 @@ public static boolean loadLogo(Identifier logoId, String modName, SplashProvider
 		ResourceTexture resourceTexture = new ResourceTexture(logoId);
 		try {
 			ResourceManager resourceManager = MinecraftClient.getInstance().getResourceManager();
-			Resource resource = resourceManager.getResource(logoId);
+			Resource resource = resourceManager.getResourceOrThrow(logoId);
```

Comment:
this api returns an Optional now, rather than refactor this block I just retained the same flow and have it throw when null